content/docs/api-reference/tls-ca-certificates-api/index.md

```diff
@@ -13,4 +13,158 @@ weight: 113
 toc: true
 ---
 
+## TLS CA Certificates API Examples
+
+Create the [bearer_token](../../setup-and-administration/getting-access-tokens-using-client-secret) and store in a file in a secure local directory with 0600 permissions.
+
+Set the URL (for example):
+
+```bash
+export URL=https://synsation.1234-5678.nodes.archivist.jitsuin.io 
+```
+
+### TLS CA Certificate Upload
+
+Define the TLS CA certificate parameters and store in `/path/to/jsonfile` (certificate field shortened for brevity):
+
+```json
+{
+    "display_name": "Some description",
+    "certificate": "-----BEGIN CERTIFICATE-----\nMIIFxDCCA6ygAwIBAgIBAjANBgkqhkiG9w0BAQsFADCBsDELMAkGA1UEBhMCVVMx\nETAPBgNV....1NF/BjDZ4wdexw==\n-----END CERTIFICATE-----\n"
+}
+```
+
+To include the PEM file content in a JSON string it must be flattened to a single line. 
+To create a single line representation of a PEM file for the archivist api, you must replace new lines with the literal string “n”. 
```

Comment:
It’s \n not just n!

Reply:
Fixed :) - that was a direct copy form the old docs!